### PR TITLE
Update Images FAQ

### PR DESCRIPTION
**Pull request by ltagliaferri**
Updates to the Images FAQ, resolves https://github.com/chainguard-dev/internal/issues/1625

### CODE REVIEW

File: content/chainguard/chainguard-images/faq.md [hunk 1]
```diff
@@ -14,35 +14,74 @@ weight: 600
...
 
-### What is an "undistro"?
+## What is an "undistro"?
 We call Wolfi an undistro because unlike a typical Linux distribution, Wolfi is a stripped-down distribution designed for the cloud-native era. Most notably, we don’t include a Linux kernel, instead relying on the environment (such as the container runtime) to provide this.
 
-### Which images are available?
-You can check which images are already available at our [Images Catalog](https://edu.chainguard.dev/chainguard/chainguard-images/reference/), through the [Chainguardy Registry](/chainguard/chainguard-images/registry/overview/), or in our [GitHub Repository](https://github.com/chainguard-images).
+## Which images are available?
+
+Chainguard Images currently support more than 100+ images, which are segmented in [three catalog tiers](#what-are-the-different-catalog-tiers-of-chainguard-images), which you can read about in the [next question](#what-are-the-different-catalog-tiers-of-chainguard-images).
```
Comment (SharpRake) [inline content/chainguard/chainguard-images/faq.md:36]:
```suggestion
Chainguard currently support more than 100 Images, which are segmented in [three catalog tiers](#what-are-the-different-catalog-tiers-of-chainguard-images), which you can read about in the [next question](#what-are-the-different-catalog-tiers-of-chainguard-images).
```
"more than 100+" felt redundant. Also, not sure about splitting the product name like this, feel free to revert it, I just thought it read a little funny to have the word "images" in the same clause

File: content/chainguard/chainguard-images/faq.md [hunk 2]
```diff
@@ -14,35 +14,74 @@ weight: 600
...
 
-### Can I simply replace my current base image with a Chainguard Image and it will work out of the box?
+## Can I simply replace my current base image with a Chainguard Image and it will work out of the box?
 Chainguard Images are designed to be minimalist, and many of them are distroless, which means they don't come with a package manager. Depending on your stack and specific dependencies, you may need to include additional software by combining development images and distroless images in a multi-stage Docker build.
 
-### How often are Chainguard Images updated?
+## How often are Chainguard Images updated?
 Chainguard Images are rebuilt every night to ensure that new package versions and security updates in upstream Wolfi are quickly applied.
 
+## Do I need to authenticate into Chainguard to use Chainguard Images?
```
Comment (SharpRake) [inline content/chainguard/chainguard-images/faq.md:83]: This whole Q+A is very nicely written!

**Review by SharpRake — APPROVED** [review]
Only one suggestion!

File: content/chainguard/chainguard-images/faq.md (hunk 1 above)
Comment (ltagliaferri) [reply content/chainguard/chainguard-images/faq.md]: I rephrased this :)

**Pull request merged**